apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.Tests/Google.Cloud.PubSub.V1.Tests.csproj

```diff
@@ -11,7 +11,6 @@
     <PackageReference Include="Google.Apis.CloudResourceManager.v1" Version="[1.57.0.2677, 2.0.0.0)" />
     <ProjectReference Include="..\..\..\tools\Google.Cloud.ClientTesting\Google.Cloud.ClientTesting.csproj" />
     <ProjectReference Include="..\Google.Cloud.PubSub.V1\Google.Cloud.PubSub.V1.csproj" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="6.0.0" />
```

Comment:
Is this deliberate? My guess is not.

Reply:
No, this looks like it was done by owlbot? https://github.com/googleapis/google-cloud-dotnet/pull/9877/commits/4080ef35c9ad1d1cdf22fa0eed1cda2a3321a19f

Reply:
Ah yes - I understand now. Will fix tomorrow and merge this then.

Reply:
No rush, thanks!